VERSION

```diff
@@ -0,0 +1,5 @@
+0
+0
+3
+dev
+0
```

Comment:
I'm not super-happy with this file. :/ Is there a way to keep the authoritative version number as a string inside the binary? I don't want someone who `go get`s this library (the preferred mechanism) to wind up with a different version than someone who grabs release binaries. Or who runs a docker version.